package.json

```diff
@@ -2,13 +2,14 @@
   "name": "ah-aquaman-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "app.js",
```

Comment:
Thank you @CryceTruly for good work, can you please change `app.js` to `index.js` Since we have an `App.js` file in a src folder. This might bring up a conflict.